### PR TITLE
Updated the setup.cfg file to contain all the metadata

### LINKED ISSUE #135: Move to setup.cfg
🐞 **Problem**

Migrate from setup.py to setup.cfg!
🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by yashrsharma44**
* This PR would ensure that all the metadata is stored in the `setup.cfg` file
* Currently, only the version number is not stored in the `setup.cfg`
##

Fixes #135 

**Review**
@shreyasbapat

### CODE REVIEW

**Comment by ritzvik** [comment]
@yashrsharma44 Check `setup.py` and `setup.cfg` in https://github.com/poliastro/poliastro. Maybe, change them on the lines of this repo.

Also, we use `black` and `isort` with each python file in the repo. They would be installed automatically if you have the dev version. Just run `black /path/to/file` on each `.py` file. Same for `isort`.

See https://docs.einsteinpy.org/en/latest/getting_started.html#development-version for installing the dev version.

Thanks for the PR @yashrsharma44

**Comment by shreyasbapat** [comment]
> @yashrsharma44 Check setup.py and setup.cfg in https://github.com/poliastro/poliastro. Maybe, change them on the lines of this repo.

No! We don't want that automation. It has caused a lot of problems :)

> This PR would ensure that all the metadata is stored in the setup.cfg file

Right!

**Comment by ritzvik** [comment]
I have reason to believe that older version of pip is being used somehow in `test_py35`.
@yashrsharma44, In `.circleci/config.yml` could you add `pip install --upgrade pip` after line 42 before `pip install numpy`. May solve the failing test, not sure.

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ jobs:
           command: |
             python3 -m venv venv
             . venv/bin/activate
+            pip install --upgrade pip
```
Comment (shreyasbapat) [inline .circleci/config.yml]: Everything must be quiet :)

Comment (yashrsharma44) [reply .circleci/config.yml]: Sure !

**Comment by shreyasbapat** [comment]
> I have reason to believe that older version of pip is being used somehow in test_py35.
@yashrsharma44, In .circleci/config.yml could you add pip install --upgrade pip after line 42 before pip install numpy

I didn't see this! No, it's not pip issue. Its a matplotlib issue! See the logs :)

**Comment by ritzvik** [comment]
> > I have reason to believe that older version of pip is being used somehow in test_py35.
> > @yashrsharma44, In .circleci/config.yml could you add pip install --upgrade pip after line 42 before pip install numpy
> 
> I didn't see this! No, it's not pip issue. Its a matplotlib issue! See the logs :)
> 
> EDIT: This time tests did run. I was looking at the old report!

The updated config file does not reflect in CircleCI

**Comment by shreyasbapat** [comment]
Correct because the update has been done in Python 3.6 instead of Python 3.5 :)

**Comment by ritzvik** [comment]
> Correct because the update has been done in Python 3.6 instead of Python 3.5 :)

Yups. @yashrsharma44 The change was requested after line 42, I guess you did it after line 74 😃

**Comment by ritzvik** [comment]
> I have reason to believe that older version of pip is being used somehow in `test_py35`.
> @yashrsharma44, In `.circleci/config.yml` could you add `pip install --upgrade pip` after line 42 before `pip install numpy`. May solve the failing test, not sure.

It worked !!! 😄

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+name = einsteinpy
+author = Shreyas Bapat
+author_email = bapat.shreyas@gmail.com
+license = MIT
+description = Python package for General Relativity
+keywords = 
+    general relativity
+    geodesics
+    relativity
+    kerr
```
Comment (shreyasbapat) [inline setup.cfg]: Can you add some more keywords? 

black hole, kerr-newman, schwarzschild

**Comment by shreyasbapat** [comment]
This will need a rebase onto the current master :)
And I will merge it quickly!

**Comment by yashrsharma44** [comment]
Hey, I am away from keyboard, would rebase it tomorrow 😅

**Comment by shreyasbapat** [comment]
Let's merge this without the merge commit! :)

**Pull request merged**